app/vault_client.py

```diff
@@ -61,55 +65,57 @@ def post(self):
         threshold = The minimum number of generated keys needed to unseal the
             vault
         '''
-        json_body = request.json
+        app.logger.debug('Init vault endpoint called')
 
-        marshal_json = marshal(json_body, _vault_init_params)
+        vault_params = marshal(request.json, _vault_init_params)
 
-        shares = marshal_json['shares']
-        threshold = marshal_json['threshold']
+        shares = vault_params['shares']
+        threshold = vault_params['threshold']
 
         if threshold > shares or shares >= 2 and threshold == 1 or shares < 1 or threshold < 1:
-            resp = json_response.create(HttpCode.bad_request.value, HttpBody.init_vault_fail_due_to_parameter.value)
+            resp = json_response.create(HttpCode.BAD_REQUEST.value, HttpBody.INIT_VAULT_BAD_REQUEST.value)
             return resp
```

Comment:
I don't think we ever need resp. It gets returned immediately everywhere
```suggestion
            return json_response.create(HttpCode.BAD_REQUEST.value, HttpBody.INIT_VAULT_BAD_REQUEST.value)
```

Reply:
Agreed.